app/src/components/market/common/categories/index.tsx

```diff
@@ -11,12 +11,16 @@ interface Props {
   name: string
   onChange?: any
   selectedCategory: string
+  setFirst: React.Dispatch<React.SetStateAction<number>>
```

Comment:
I think this type could also be written as `(n: number) => void`  in other places is used with that convention.